frontend/src/modules/EditZing/Components/StudentGrid.tsx

```diff
@@ -80,17 +65,43 @@ export const StudentGrid = ({
               submissionTime.getDate()
             }
           >
-            <StyledStudentText>{student.name}</StyledStudentText>
+            <Box
+              sx={{
+                display: 'flex',
+                flexFlow: 'row nowrap',
+                gap: '13px',
+                position: 'relative',
+              }}
+            >
+              <Box sx={{ width: '90%' }}>
+                <Typography sx={{ fontWeight: '800', fontSize: '0.875rem' }}>
+                  {student.name}
+                </Typography>
+                <Typography sx={{ fontWeight: '400', fontSize: '0.875rem' }}>
+                  {student.email.replace('@cornell.edu', '')}
+                </Typography>
+                <Typography sx={{ fontWeight: '400', fontSize: '0.875rem' }}>
+                  {student.year}
+                </Typography>
+              </Box>
+
+              <Checkbox
```

Comment:
You can try adding `disableRipple` prop to this checkbox because the current ripple looks odd b/c of how close it is to the edge of the card in the designs